Gemfile

```diff
@@ -8,47 +8,66 @@ end
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.1.5'
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3'
 # Use Puma as the app server
 gem 'puma', '~> 3.7'
 # Use SCSS for stylesheets
 gem 'sass-rails', '~> 5.0'
+# Bootstrap sass
+gem 'bootstrap'
```

Comment:
Try to consider what gems you'll need for an API that only returns JSON. Bootstrap and jQuery are likely to go unused and just add to the time it takes the application to load.

Reply:
Right, I just copied it from my prior project. I'll remove the remaining gems I'm not using.